tests/sentry/api/endpoints/test_organization_teams.py

```diff
@@ -125,36 +130,27 @@ def test_valid_params(self):
         ).exists()
 
     def test_without_slug(self):
-        self.login_as(user=self.user)
+        resp = self.get_valid_response(self.organization.slug, name="hello world", status_code=201)
 
-        resp = self.client.post(self.path, data={"name": "hello world"})
-        assert resp.status_code == 201, resp.content
         team = Team.objects.get(id=resp.data["id"])
         assert team.slug == "hello-world"
 
     def test_without_name(self):
-        self.login_as(user=self.user)
+        resp = self.get_valid_response(self.organization.slug, slug="example-slug", status_code=201)
 
-        resp = self.client.post(self.path, data={"slug": "example-slug"})
-        assert resp.status_code == 201, resp.content
         team = Team.objects.get(id=resp.data["id"])
         assert team.slug == "example-slug"
         assert team.name == "example-slug"
 
     def test_duplicate(self):
-        self.login_as(user=self.user)
-
-        resp = self.client.post(self.path, data={"name": "hello world", "slug": "foobar"})
-
-        assert resp.status_code == 201, resp.content
-
-        resp = self.client.post(self.path, data={"name": "hello world", "slug": "foobar"})
-
-        assert resp.status_code == 409, resp.content
+        self.get_valid_response(
+            self.organization.slug, name="hello world", slug="foobar", status_code=201
+        )
+        self.get_valid_response(
```

Comment:
Should the method be `get_response`? The name `get_valid_response()` and expecting a 409 seem at odds.

Reply:
`get_response` is already used, but I've created two new functions (`get_success_response` and `get_error_response`) that should make the tests clearer.